Segments/Bloodlands.mapproj

```diff
@@ -46719,7 +46719,7 @@
       </tile>
       <tile x="18" y="-67">
         <component type="FloorComponent">
-          <ground>12</ground>
```

Comment:
this looks like the y got typed in the wrong field maybe?

Reply:
Yep, for whatever reason when you type in exclusion area with a component up, it types in component as well? 

![image](https://user-images.githubusercontent.com/90510109/167860284-bb21e776-a3ed-4598-bc78-1abed94eb84a.png)